Assets/Scripts/BowlingBall.cs

```diff
@@ -1,151 +0,0 @@
-﻿using System;
-using System.Collections;
-using System.Collections.Generic;
-using System.Security.AccessControl;
-using UnityEngine;
-using UnityEngine.EventSystems;
-using UnityEngine.Experimental.UIElements;
-using UnityEngine.XR.WSA;
```

Comment:
You have all kinds of crap in the usings here. It's not really a problem, but it's nice to remove unused stuff. Tools will often have a way of doing this automatically.